app/src/main/java/com/ncnf/storage/FileStore.java

```diff
@@ -0,0 +1,77 @@
+package com.ncnf.storage;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.widget.ImageView;
+
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.storage.FirebaseStorage;
+import com.google.firebase.storage.StorageReference;
+import com.google.firebase.storage.UploadTask;
+import com.ncnf.database.DatabaseResponse;
+
+import java.io.ByteArrayOutputStream;
+import java.util.concurrent.CompletableFuture;
+
+public class FileStore {
+
+    private final long MAX_SIZE = 10 * 1024 * 1024;
```

Comment:
Maybe define a project global variable ?

Reply:
This variable makes sense in the context of the FileStore. If someone needs it, we can make it public.